cmd/protoc-gen-gcli/testprotos/kiosk/kiosk.proto

```diff
@@ -0,0 +1,141 @@
+// Copyright (c) 2018, Google LLC.
```

Comment:
I assume this was copied from somewhere?

Reply:
This proto was taken from github.com/googleapis/kiosk, yes